lib/event-stream/format-message.js

```diff
@@ -0,0 +1,166 @@
+var crypto = require('../../lib/util').crypto;
+var Int64 = require('./int64').Int64;
+var toBuffer = require('./to-buffer').toBuffer;
+var allocBuffer = require('./alloc-buffer').allocBuffer;
+
+function formatMessage(message) {
```

Comment:
This name is a bit confusing to me. `buildMessage` is probably a better name.

Reply:
Sure, I don't have a strong preference either way.